commcare_connect/templates/opportunity/opportunity_list.html

```diff
@@ -45,15 +51,23 @@ <h1> Opportunities
                    href="{% url 'opportunity:detail' org_slug=request.org.slug pk=opportunity.id %}">
                   <span class="bi bi-eye"></span><span class="d-none d-md-inline">&nbsp;View</span>
                 </a>
+              {% if request.org_membership.is_viewer %}
```

Comment:
Similar to https://github.com/dimagi/commcare-connect/pull/329/files#r1646916211

We should try to have a single HTML and disable condition within it, rather than branching on the condition to different HTML